### PR TITLE
Linters configuration

### PR DESCRIPTION
**Pull request by MattGomb**
We are creating a template project to learn how to configure some basic linters. Practicing with webhint and stylelint.

### CODE REVIEW

**Comment by MattGomb** [comment]
finished all the linter fixes, so the checks all pass now.

File: README.md [hunk 1]
```diff
@@ -2,18 +2,11 @@
...
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+html, and css
 
 
 ## Getting Started
```
Comment (Ismailco) [inline README.md:11]: - [x] Kindly customize the getting start section by adding some steps to how the user can work with the project.
   e.g 
  ```
   git clone repo.git
   cd repo
   ```

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @MattGomb, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️

- [x] Branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly pay attention to this in the future projects

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MattGomb** [comment]
Hi Ismail,

I am not sure if you will get this, or how this can go. I tried to rename
my branch, now i think it is broken, and i don't know how to fix it. It is
showing 15 thousand (!!!) changes in the main branch, i have no idea what
happened.
[image: image.png]

I would appreciate suggestions.....

Thanks,
Matt

**Comment by Ismailco** [comment]
@MattGomb, it's not required to change the branch name, I commented `kindly pay attention to this in the future projects` not modify it in this branch because it will be closed if you do

**Comment by MattGomb** [comment]
So how do i fix this..? I changed the readme file, but the pull request is closed now.

**Comment by MattGomb** [comment]
@Ismailco So what shoud i do? I reopened it, but it seems like i can't update the readme file in VSC anymore..?

**Review by levy002 — APPROVED** [review]
## Hi, @MattGomb :wave:  
### I'm Levy, your PR reviewer.

## Status: APPROVED :fire: 

Excellent work. I really like your communication with the previous reviewer. You were so professional. :sunglasses: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :sunglasses:  Congratulations! 🎉 🎉 🎉 

5SW

## Highlights :muscle:
- You used the correct Github Flow. :heavy_check_mark: 
- Professional README. :heavy_check_mark: 
- Linters are green :heavy_check_mark: 
- Good commit history :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to write your commit message in imperative tense and capitalized it in future projects. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

- [OPTIONAL ] I think it would be better in future projects to add a good PR title and PR description. It makes your repo looks more professional. :wink: 

   - Example of good PR
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @levy002 me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MattGomb** [comment]
Thanks for the feedback!  I'll make sure to review that lesson!

**Pull request merged**